nvdaHelper/UIARemote/UIARemote.cpp

```diff
@@ -0,0 +1,154 @@
+/*
+This file is a part of the NVDA project.
+URL: http://www.nvaccess.org/
+Copyright 2021-2022 NV Access Limited
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License version 2.0, as published by
+    the Free Software Foundation.
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+This license can be found at:
+http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+*/
+
+#define _SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING

```

Comment:
Could you add a comment with an example warning and where you see it? Makes it easier to work out when/if this is no longer required.